stock_request/views/stock_request_report.xml

```diff
@@ -0,0 +1,58 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!--
+    Copyright 2021 Creu Blanca
+    License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).
+-->
+<odoo>
+    <menuitem
+        id="menu_stock_request_report_root"
+        name="Reports"
+        sequence="200"
+        parent="stock_request.menu_stock_request_root"
+    />
+    <record id="stock_request_pivot" model="ir.ui.view">
+        <field name="name">stock.request.pivot</field>
+        <field name="model">stock.request</field>
+        <field name="arch" type="xml">
+            <pivot
+                string="Stock Requests"
+                disable_linking="True"
+                display_quantity="True"
+            >
+                <field name="product_id" type="row" />
+                <field name="warehouse_id" type="col" />
+                <field name="location_id" type="col" />
+            </pivot>
+        </field>
+    </record>
+    <record id="stock_request_pivot_search" model="ir.ui.view">
+        <field name="name">stock.request.pivot.search</field>
+        <field name="model">stock.request</field>
+        <field name="inherit_id" ref="stock_request.stock_request_search" />
+        <field name="arch" type="xml">
+            <filter name="current_requests" position="after">
+                <separator />
+                <filter
+                    string="Date"
+                    name="month"
```

Comment:
Date or month?
```suggestion
                    string="Date"
                    name="month"
```

Reply:
I have changed all to date, but if you consider that "month" is a better option I changed it again :blush: